integration_test/common/testdata/integration-metadata_expected-metric_onetrue_two_represenetative/input.yaml

```diff
@@ -0,0 +1,32 @@
+app_url: "https://example.google.com/"
+short_name: OpsAgent
+long_name: Cloud OpsAgent
+description: |-
+  Example description
+minimum_supported_agent_version:
+  metrics: 1.0.0
+  logging: 1.0.0
+supported_app_version: ["Example0"]
+expected_metrics:
+- type: example.googleapis.com/example.example1
+  value_type: INT64
+  kind: GAUGE
+  monitored_resource: gce_instance
+  labels:
+    broker: .*
+    destination: .*
+  representative: true
+- type: example.googleapis.com/example.example1
```

Comment:
`example2`?

Reply:
Changed in: [42da351](https://github.com/GoogleCloudPlatform/ops-agent/pull/673/commits/42da351c6e2089dcedf53a561fe57305c67ce00c)